### PR TITLE
fix a typo

### PR DESCRIPTION
**Pull request by dooglus**
The opening pop-up has a typo in it.

### CODE REVIEW

File: js/languages/de.json [hunk 1]
```diff
@@ -516,7 +516,7 @@
   "onboarding": {
     "intro": "OpenBazaar Konfiguration",
     "Introduction": "Einf&uuml;hrung",
-    "IntroductionBody": "OpenBazaar is a peer-to-peer social market. It's like combining eBay&trade;, Twitter&trade; and BitTorrent into one. Only, there are no fees or restrictions and OpenBazaar is open-source. \n\n Please note that it may look and feel a bit different than what you're use to, so please be patient as you adjust.",
+    "IntroductionBody": "OpenBazaar is a peer-to-peer social market. It's like combining eBay&trade;, Twitter&trade; and BitTorrent into one. Only, there are no fees or restrictions and OpenBazaar is open-source. \n\n Please note that it may look and feel a bit different than you are used to, so please be patient as you adjust.",
```
Comment (jjeffryes) [inline js/languages/de.json:519]: This isn't a typo, and doesn't need to be changed.

File: js/languages/en-US.json [hunk 2]
```diff
@@ -538,7 +538,7 @@
   "onboarding": {
     "intro": "OpenBazaar Configuration",
     "Introduction": "Introduction",
-    "IntroductionBody": "OpenBazaar is a peer-to-peer social market. It's like combining eBay&trade;, Twitter&trade; and BitTorrent into one. Only, there are no fees or restrictions and OpenBazaar is open-source. \n\n Please note that it may look and feel a bit different than what you're use to, so please be patient as you adjust.",
+    "IntroductionBody": "OpenBazaar is a peer-to-peer social market. It's like combining eBay&trade;, Twitter&trade; and BitTorrent into one. Only, there are no fees or restrictions and OpenBazaar is open-source. \n\n Please note that it may look and feel a bit different than you are used to, so please be patient as you adjust.",
```
Comment (jjeffryes) [inline js/languages/en-US.json:541]: This isn't a typo, and doesn't need to be changed.

**Comment by jjeffryes** [comment]
The text changed was fine in it's original form.

**Comment by dooglus** [comment]
It is "used to" not "use to".

"different than what you're use to" isn't grammatical.

See http://www.grammar.cl/rules/used-to-use-to.htm for example:

> To be used to = accustomed to something (normally out of habit or experience)

Or you could have the first thing the user sees when they run your program contain a grammatical error. It is your call. Sorry for trying to make things better.

**Comment by dooglus** [comment]
> The text changed was fine in **it's** original form.

You don't seem to be very good at English grammar. Maybe you should ask somebody who knows about these things.

"It's" is a contraction of "it is", so you wrote "was fine in **it is** original form". Get it?

**Pull request closed without merging**